README.md

```diff
@@ -112,6 +112,66 @@ $ make nodelink-controller
      new instance is created. Followed by new node joining the cluster
      and turning in `Ready` state.
 
+## Creating machines
+
+You can create a new machine by applying a manifest representing an instance of the machine CRD:
+```
+apiVersion: machine.openshift.io/v1beta1
```

Comment:
Link to authoritative source instead of inline?  This information is possible to go stale.